icons/layers.svg

```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <polygon points="12 2 2 7 12 12 22 7 12 2" />
-  <polyline points="2 17 12 22 22 17" />
-  <polyline points="2 12 12 17 22 12" />
+  <path d="M22 7c0-.3-.2-.6-.6-.8l-7.6-3.8a4.78 4.78 0 0 0-3.6 0L2.6 6.2a.94.94 0 0 0 0 1.6l7.6 3.8a4.77 4.77 0 0 0 3.6 0l7.6-3.8c.4-.2.6-.5.6-.8" />
+  <path d="m2 12.5 8.2 4.1a4.77 4.77 0 0 0 3.6 0l8.2-4.1" />
+  <path d="m2 17.5 8.2 4.1a4.77 4.77 0 0 0 3.6 0l8.2-4.1" />
```

Comment:
Yay? ![icon (55)](https://github.com/lucide-icons/lucide/assets/25524993/23ece982-7424-4426-b031-f5f525e90efe)
```suggestion
  <path d="m6 14.5-3.4 1.7a.83.83 0 0 0 0 1.6l7.6 3.8a4.14 4.14 0 0 0 3.58 0l7.62-3.8a.83.83 0 0 0 0-1.6L18 14.5" />
  <path d="m6 9.5-3.4 1.7a.83.83 0 0 0 0 1.6l7.6 3.8a4.14 4.14 0 0 0 3.58 0l7.62-3.8a.83.83 0 0 0 0-1.6L18 9.5" />
  <path d="M13.8 2.4a4.14 4.14 0 0 0-3.58 0L2.6 6.2a.83.83 0 0 0 0 1.6l7.6 3.8a4.14 4.14 0 0 0 3.58 0l7.62-3.8a.83.83 0 0 0 0-1.6Z" />
```

Reply:
Fine with me. I also had this in drafts, but figured maybe it might be better to leave the choice between styles… @karsa-mistmere What do you think?

Reply:
> Yay?

@jguddas Added as `layers-3` (which actually works out nicely in this case, as `layers-2` is literally 2 layers, and `layers-3` has 3… 👌🏼)

Reply:
Same as `layers-3`:
```suggestion
  <path d="m12.83 2.18a2 2 0 0 0-1.66 0L2.6 6.08a1 1 0 0 0 0 1.83l8.58 3.91a2 2 0 0 0 1.66 0l8.58-3.9a1 1 0 0 0 0-1.83Z" />
  <path d="m22 17.65-9.17 4.16a2 2 0 0 1-1.66 0L2 17.65" />
  <path d="m22 12.65-9.17 4.16a2 2 0 0 1-1.66 0L2 12.65" />
```